OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -84,6 +84,9 @@
       <HintPath>..\thirdparty\download\MaxMind.Db.dll</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="SmarIrc4net">
+      <HintPath>..\thirdparty\download\SmarIrc4net.dll</HintPath>
```

Comment:
If we ever get to extracting a dedicated server into a single .exe, this would probably mean that it would depend on this .dll, right?